scripts/setup-devnet.sh

```diff
@@ -13,37 +13,37 @@ killall -v swapd
 pkill -e -f swapcli
 
 echo "start ganache"
-"$(npm config get prefix)/bin/ganache" --deterministic --accounts=50 --miner.blockTime=1 &> "${LOG_DIR}/ganache.log" &
+"$(npm config get prefix)/bin/ganache" --deterministic --accounts=50 --miner.blockTime=1 &>"${LOG_DIR}/ganache.log" &
```

Comment:
I haven't used this script and don't entirely know what it is for, so I didn't try to simplify it.  All the changes here were from the formatter or to make the linter happy.

Reply:
feel free to delete this script, it's a duplicate of what setup-env.sh was

Reply:
Deleted.  Thanks!